.github/workflows/CI.yml

```
@@ -263,3 +263,30 @@ jobs:
         with:
           command: rustdoc
           args: --features full,ffi -- --cfg docsrs --cfg hyper_unstable_ffi -D broken-intra-doc-links
+
+  check-external-types:
+    name: Check exposed types
+    needs: [style, test]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

We should use the latest version of this action. Older versions of this action depend on node versions that are deprecated in Github's runners.

```suggestion
        uses: actions/checkout@v3
```